### ISSUE TITLE
Chapter 6, page 203

### ISSUE THREAD

**Issue by GermanCM** [rasbt/python-machine-learning-book-2nd-edition#113]
Dear professor Sebastian,

I think there is no need to retrain the best estimator with the train set after using 'GridSearchCV', since the 'GridSearchCV' class already implements, by default (with the 'refit' param = True by default), a model re-training on the whole train dataset with the best found hyperparameters (based on the defined metric). 
So in your example in this page, we could implement directly:
clf = gs.best_estimator_
--> this line would not be necessary: clf.fit(X_train, y_train)
print('Test accuracy: %.3f' % clf.score(X_test, y_test))
Test accuracy: 0.974

I hope I explained this clearly. 
Best regards and thanks for your excellent book.

Germán

**Comment by rasbt** [comment]
Hi Germán,

you are absolutely right. I think I listed this as two independent steps to make the general workflow/concept more clear (independent of scikit-learn). However, you can set `refit=True` to achieve the same effect. Computationally, it should make no difference in terms of how expensive it is to run the code. Thanks for the note though, I should mention this.

**Comment by rasbt** [comment]
Oh I see that the ` clf.fit(X_train, y_train)` is completely redundant since it's `refit=True` by default as you correctly pointed out!

**Comment by GermanCM** [comment]
Thanks dear Sebastian for including a note about this in your third edition, I will close the issue :)
